client/src/lib/fetchFunctions.ts

```diff
@@ -0,0 +1,25 @@
+import { Entry } from './types';
+
+export async function createEntry(entryInput: Entry) {
+  const req = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(entryInput),
+  };
+  const res = await fetch('/api/entries', req);
+  if (!res.ok) {
+    throw new Error(`fetch Error ${res.status}`);
+  }
+  return await res.json();
+}
+
+export async function searchUnsplash(search: string) {
+  const res = await fetch('/api/key');
```

Comment:
If you're worried about calling this too often (though it's most likely not an issue), you could put the key into a global variable in this file. Then make this call only if it's undefined. The global variable is safe from prying and I'm guessing the key will never change, so it seems wasteful to make this call repeatedly.